dask/array/stats.py

```diff
@@ -42,16 +43,27 @@
     raise ImportError("`dask.array.stats` requires `scipy` to be installed.") from e
 from scipy import special
 from scipy.stats import distributions
-from scipy.stats.stats import (
-    F_onewayResult,
-    KurtosistestResult,
-    NormaltestResult,
-    Power_divergenceResult,
-    SkewtestResult,
-    Ttest_1sampResult,
-    Ttest_indResult,
-    Ttest_relResult,
-)
+
+# copied from https://github.com/scipy/scipy/blob/v1.8.0/scipy/stats/_stats_py.py since
+# these are all private after v1.8.0
+F_onewayResult = namedtuple("F_onewayResult", ("statistic", "pvalue"))
+KurtosistestResult = namedtuple("KurtosistestResult", ("statistic", "pvalue"))
+NormaltestResult = namedtuple("NormaltestResult", ("statistic", "pvalue"))
+Power_divergenceResult = namedtuple("Power_divergenceResult", ("statistic", "pvalue"))
+SkewtestResult = namedtuple("SkewtestResult", ("statistic", "pvalue"))
+Ttest_1sampResult = namedtuple("Ttest_1sampResult", ("statistic", "pvalue"))
+Ttest_indResult = namedtuple("Ttest_indResult", ("statistic", "pvalue"))
+Ttest_relResult = namedtuple("Ttest_relResult", ("statistic", "pvalue"))
```

Comment:
Hi @jsignell 

Thanks for this!

I got the impression from the `scipy` error message:
```python
DeprecationWarning: Please use `F_onewayResult` from the `scipy.stats` namespace, 
the `scipy.stats.stats` namespace is deprecated.
```
that these attributes were not really meant to be private, which is why I reported the issue.  But your solution certainly allows `dask` to use version 1.8.0 while `scipy` figures this out.

Reply:
Yeah it's not totally clear to me whether these are intentionally being made private or not. I am also not sure what the intention is with `MatrixLinearOperator`.